src/beautifiers/exfmt.coffee

```diff
@@ -0,0 +1,21 @@
+###
+Requires https://github.com/lpil/exfmt
+###
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Exfmt extends Beautifier
+  name: "exfmt"
+  link: "https://github.com/lpil/exfmt"
+  isPreInstalled: false
```

Comment:
> Full command to see version is `mix help exfmt` 

Then I would expect something like:

```
cmd: 'mix',
version: {
        args: ['help', 'exfmt'],
        parse: (text) -> text.match(/exfmt-(\d+\.\d+\.\d+)/)[1]
      }
```

The full command being: `${cmd} ${version.args.join(' '}`

Then you test it by reloading Atom and trying to beautify. 
When installed, if the version does not correctly parse it will throw an error.
Try to uninstall `exfmt` and try again, see if it throws error as you expect. So you know it works when it should work, and error when it should not (i.e. not installed).